src/Executor/Executor.php

```diff
@@ -34,29 +35,37 @@ public function createRuntime(
         string $functionId, 
         string $deploymentId, 
         string $projectId, 
-        string $source, 
-        array $vars, 
+        string $source,
         string $runtime, 
         string $baseImage,
-        array $commands
+        bool $remove = false,
+        string $entrypoint = '',
+        string $workdir = '',
+        string $destination = '',
+        string $network = '',
+        array $vars = [],
+        array $commands = []
     ) {
         $route = "/runtimes";
         $headers = [
             'content-type' => 'application/json',
-            'x-appwrite-project' => $projectId,
             'x-appwrite-executor-key' => App::getEnv('_APP_EXECUTOR_SECRET', '')
         ];
         $params = [
             'runtimeId' => "$projectId-$deploymentId",
             'source' => $source,
-            'destination' => APP_STORAGE_BUILDS . "/app-$projectId",
-            'vars' => $vars,
+            'destination' => $destination,
             'runtime' => $runtime,
             'baseImage' => $baseImage,
+            'entrypoint' => $entrypoint,
+            'workdir' => $workdir,
+            'network' => empty($network) ? App::getEnv('_APP_EXECUTOR_RUNTIME_NETWORK', 'openruntimes') : $network,
```

Comment:
is default `openruntimes` fine? This being Appwrite, may be we can keep `appwriteruntimes`?

Reply:
Good point. Let me do that 👍